src/directives/games-list/index.jade

```diff
@@ -0,0 +1,12 @@
+md-list(layout='row', layout-margin, layout-padding, layout-align='center stretch')
```

Comment:
I've used a toolset which uses angular - but includes a ton of boilerplate UI: https://material.angularjs.org/latest/getting-started

It's faster to write - and you'll see Jade really shine here. As the Material Design HTML elements kinda look more like a hierarchy of function calls - at least the ones that do anything special.... You'll see what the hell i'm talking about after reading the code a bit more, check my code in `src/directives/` ... 🏆